drivers/ng_at86rf2xx/ng_at86rf2xx_getset.c

```diff
@@ -235,6 +235,20 @@ void ng_at86rf2xx_set_txpower(ng_at86rf2xx_t *dev, int16_t txpower)
 #endif
 }
 
+uint8_t ng_at86rf2xx_get_max_retries(ng_at86rf2xx_t *dev)
+{
+    return (ng_at86rf2xx_reg_read(dev, NG_AT86RF2XX_REG__XAH_CTRL_0) >> 4);
+}
+
+void ng_at86rf2xx_set_max_retries(ng_at86rf2xx_t *dev, uint8_t max)
+{
+    max = (max > 7) ? 7 : max;
+    uint8_t tmp = ng_at86rf2xx_reg_read(dev, NG_AT86RF2XX_REG__XAH_CTRL_0);
+    tmp = (tmp & (NG_AT86RF2XX_XAH_CTRL_0__MAX_CSMA_RETRIES |
```

Comment:
In ``RIOT/drivers/ng_at86rf2xx/ng_at86rf2xx_getset.c`` (l249):
>            case NG_AT86RF2XX_OPT_CSMA:
                DEBUG("[ng_at86rf2xx] opt: enabling CSMA mode (NOT IMPLEMENTED)\n");
                /* TODO: en/disable csma */
                break;
Then this is just the get/set functions which are not implemented?